mlrun/model_monitoring/controller.py

```diff
@@ -71,14 +71,20 @@ def _get_last_analyzed(self) -> Optional[int]:
         except HttpResponseError as err:
             logger.warn(
                 "Failed to get the last analyzed time for this endpoint and application, "
-                "as this is probably the first time this application is running. "
-                "Using the first request time instead.",
+                "as this is probably the first time this application is running. ",
```

Comment:
Let's check and exception (or introduce exception types to v3io-py)  and log an error only if it's not 404/not found.
On the other hand, send info-sev log when we init the last_analyzed (see other comment)